src/core/core.js

```diff
@@ -234,7 +235,8 @@ export default class Core {
         geolocation: this.globalStorage.getState(StorageKeys.GEOLOCATION),
         skipSpellCheck: this.globalStorage.getState('skipSpellCheck'),
         queryTrigger: this.globalStorage.getState('queryTrigger'),
-        sessionTrackingEnabled: this.globalStorage.getState(StorageKeys.SESSIONS_OPT_IN)
+        sessionTrackingEnabled: this.globalStorage.getState(StorageKeys.SESSIONS_OPT_IN),
+        context: this.persistentStorage.get(StorageKeys.API_CONTEXT)
```

Comment:
currently we try to use globalStorage instead of persistentStorage where possible. Think this helps separate responsibility a bit, and eventually we want to turn persistentStorage (or however we persist session data) into an implementation detail of globalStorage.

Reply:
Sure. So the situation here is that we definitely need the context in the persistentStorage. We want people to be able to copy & paste a link with the same context and get the same experience. I can change this to use globalStorage.getState instead of persistentStorage.get, I just thought it was weird that we're /relying/ on persistentStorage being added to globalStorage in ANSWERS.init instead of getting the value from the source (persistentStorage directly). AFAIK nothing should be changing the globalStorage version of this separate from the persistentStorage version of this, and I only added it to globalStorage  in `setContext` because it was already added to globalStorage by the `globalStorage.setAll(persistentStorage.getAll())`

If the context is that we will eventually get rid of persistentStorage and move toward one whole globalStorage, I am fine with removing this getter.

Reply:
yeah we wanna avoid persistent storage like its the plague. in general we want one way of doing things

Reply:
Updated!